tests/mr_slice.rs

```diff
@@ -155,6 +156,8 @@ mod remote_mr_slice_overbound {
 
         async fn client(rdma: Rdma) -> io::Result<()> {
             let rmr = rdma.receive_remote_mr().await.unwrap();
+            // wait for the agent thread to send all reponses to the remote.
```

Comment:
What's the error if we don't wait for 1s?

Reply:
348c77389f9349cf8266600766d9530956c1caf3
> Sleep after receive. Wait for the agent thread to send all reponses to the remote.
Direct exiting may cause the thread killed before the response is issued, causing remote timeout.